File: Python/백준/Silver/27035. Bovine Ballroom Dancing/Bovine Ballroom Dancing.py
a = int(input())
aa = []
bb = []
for i in range(a):
    aa.append(int(input()))
for i in range(a):
    bb.append(int(input()))
aa.sort()
bb.sort()
s = 0
for i in range(a):
    s+=abs(aa[i]-bb[i])
print(s)

File: Java/백준/Silver/27035. Bovine Ballroom Dancing/Bovine Ballroom Dancing.java
import java.util.*;
import java.lang.Math;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int a = in.nextInt();
        int[] aa = new int[a];
        int[] bb = new int[a];
        for (int i=0;i<a;i++){
            aa[i] = in.nextInt();
        }
        for (int i=0;i<a;i++){
            bb[i]= in.nextInt();
        }
        int sum = 0;
        Arrays.sort(aa);
        Arrays.sort(bb);
        for (int i=0;i<a;i++){
            sum+=Math.abs(aa[i]-bb[i]);
        }
        System.out.println(sum);
    }
}